modules/qt/hm.nix

```diff
@@ -0,0 +1,63 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}: {
+  options.stylix.targets.qt = {
+    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    iconThemeName = lib.mkOption {
+      description = "Default QT Icons";
+      type = lib.types.str;
+      default = "ePapirus-Dark";
+    };
+  };
+
+  config = lib.mkIf config.stylix.targets.qt.enable (let
+    cfg = config.stylix.targets.qt;
+    kvconfig = config.lib.stylix.colors {
+      template = ./kvconfig.mustache;
+      extension = ".kvconfig";
+    };
+    svg = config.lib.stylix.colors {
+      template = ./kvantum-svg.mustache;
+      extension = "svg";
+    };
+    kvantumPackage = pkgs.runCommandLocal "base16-kvantum" {} ''
+      mkdir -p $out/share/Kvantum/Base16Kvantum
```

Comment:
Preferably use long flags:

```suggestion
      mkdir --parents $out/share/Kvantum/Base16Kvantum
```

Reply:
mkdir -p is used in a lot more places but i will make the change
![image](https://github.com/danth/stylix/assets/52386117/104045a5-c232-411a-a5cd-b495366fa2df)